### PR TITLE
[Cardio2e] Added as new binding

### PR DESCRIPTION
**Pull request by magd1978**
Note: The README.md file was already included in the previous merged pull-request #5599. I've included it again by mistake.

### CODE REVIEW

File: features/openhab-addons/src/main/feature/feature.xml [hunk 1]
```diff
@@ -147,6 +147,13 @@
     <configfile finalname="${openhab.conf}/services/caldavPersonal.cfg" override="false">mvn:${project.groupId}/openhab-addons-external/${project.version}/cfg/caldavPersonal</configfile>
   </feature>
 
+  <feature name="openhab-binding-cardio2e" description="Cardio IIé binding" version="${project.version}">
```
Comment (9037568) [inline features/openhab-addons/src/main/feature/feature.xml]: Per the convention, the name here should be `openhab-binding-cardio2e1`.

Comment (magd1978) [reply features/openhab-addons/src/main/feature/feature.xml]: Requested change is made.

**Comment by magd1978** [comment]
Thanks to you and the entire openHAB team!

**Comment by kaikreuzer** [comment]
FTR, this PR broke the distro build (https://openhab.ci.cloudbees.com/job/openHAB-Distribution/1315/) as the binding is not built at all. I have created https://github.com/openhab/openhab1-addons/pull/5628, which should hopefully fix this.

**Pull request merged**